.github/workflows/e2e-manual-icad.yaml

```diff
@@ -12,6 +12,7 @@ on:
         options:
           - TestInterchainAccountsTestSuite
           - TestIncentivizedInterchainAccountsTestSuite
+          - TestUpgradeV6TestSuite
```

Comment:
I believe this should be `TestUpgradeTestSuit` now that both upgrade tests are on the same test suite.

Reply:
Done